src/pages/landingPage/LandingPage.js

```diff
@@ -1,140 +1,175 @@
+              fresh ingredients and a taste of the unexpected. Our artisan chefs
+              create crave-worthy classics alongside a steady rotation of
+              seasonal offerings that take advantage of naturally ripe fruits
+              and vegetables so their flavor is full and their nutrients, rich.
+              Creative, hand-crafted beverages built around seasonality, quality
+              ingredients and simple preparations complement every occasion from
+              brunch, lunch, dinner and, of course, dessert. Always room for
+              dessert.
+            </p>
+            <button className="button">Learn More</button>
+          </div>
+        </div>
+      </section>
+      <section className="landingPage-ad">
+        <p>Looking for the best affordable hospitality in town?</p>
+        <button className="button">Book Now</button>
+      </section>
+
+      <section className="landingPage-order">
+        <h2>ORDER AND DELIVERY</h2>
+        <p className="text_two">Our delivery service is top notch</p>
+        <div className="card-container">
+          {data.map((item, index) => {
+            return <Card key={index} {...item} />;
+          })}
+        </div>
+      </section>
+
+      <section className="landingPage-menu">
+        <h2>MENU</h2>
+        <p className="text_two">Choose from our most popular menu</p>
+        {
+          <div className="container ">
+            <div className="row px-0 mx-0">
+              {props.loading ? (
+                <div className="col-md-4">
+                   <div className="dish">
+                    <div className="dish-image ">
+                      <Skeleton width={'100%'} height={200} style={{margin:"auto"}} />
                     </div>
+                    <h2 className="mt-4">
+                      <Skeleton/>
+                    </h2>
+                    <p className="text-left">
+                      <Skeleton count={3} />
+                    </p>
+                    <button className="bg-light"><Skeleton/></button>
+                  </div>
                 </div>
-            </section>
-            <section className='landingPage-ad'>
-                <p>Looking for the best affordable hospitality in town?</p>
-                <button className="button">Book Now</button>
-            </section>
-            
-            <section className='landingPage-order'>
-                <h2>ORDER AND DELIVERY</h2>
-                <p className="text_two">Our delivery service is top notch</p>
-                <div className='card-container'>
-                    {
-                        data.map((item, index) => {
-                            return (
-                                <Card key={index} {...item} />
-                            )
-                        })
-                    }
-                </div>
-            </section>
+              ) : (
+                props.popularMenu?.map((dish, index) => {
```

Comment:
```suggestion
                props.popularMenu?.slice(6).map((dish, index) => {
```
can we slice this popularMenu array to 6 as suggest above

Reply:
is there any reason why we would want that?

Reply:
Yes... the landing page should not have plenty things in it... People should go to menu to view plenty menu